README.md

```diff
@@ -100,6 +100,7 @@ Finally, enable all of the rules that you would like to use.  Use [our preset](#
 * [react/no-redundant-should-component-update](docs/rules/no-redundant-should-component-update.md): Prevent usage of `shouldComponentUpdate` when extending React.PureComponent
 * [react/no-render-return-value](docs/rules/no-render-return-value.md): Prevent usage of the return value of `React.render`
 * [react/no-set-state](docs/rules/no-set-state.md): Prevent usage of `setState`
+* [react/no-static-typos](docs/rules/no-static-typos.md): Prevent casing typos when declaring static class properties.
```

Comment:
I suggested this also earlier in the PR: https://github.com/yannickcr/eslint-plugin-react/pull/1209#issuecomment-303517683

I will look into the similar issue regarding lifecycle methods. Would be useful to have it there as well, as I never remember if it was `prevProps`, `nextProps`, `prevState` or `nextState` (which one and in which order).